### ISSUE TITLE
browserLanguageDetection without fallback

### ISSUE THREAD

**Issue by benpetsch** [isaachinman/next-i18next#82]
**Expected behaviour:**

Browser language is: 'fr'
Request for `http://localhost:3001/faq` should be redirected to `http://localhost:3001/fr/faq`.

Because  'fr' is not present within 'defaultLanguage' or 'otherLanguages' configuration, I expect a fallback to 'fallbackLng'. This should result in `http://localhost:3001/en-us/faq` or `http://localhost:3001/faq` (with en-us language).

**Actual behaviour:**

Browser language is: 'fr'
Request for `http://localhost:3001/faq` is redirected to `http://localhost:3001/fr/ERROR_PAGE` with error message: `Error: Invalid configuration: Current language is not included in all languages array`

Any user with an non-default ('defaultLanguage' or 'otherLanguages') won't be able to use the website.

---

My config:
```
{
  otherLanguages: ['an', 'de', 'en'],
  localeSubpaths: true,
  defaultLanguage: 'en-us',
  browserLanguageDetection: true,
  fallbackLng: 'en-us'
}
```

**Comment by isaachinman** [comment]
Secondly, this is a very important point - good catch. I am working on a fix as we speak.

**Comment by isaachinman** [comment]
Fixed with 50bf6c8. Released in v0.17.0.
